src/components/App.js

```diff
@@ -3,13 +3,11 @@ import '../App.css';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
 
-function App() {
+export default function App() {
   return (
-    <div>
+    <div className="calc_container">
       <Display />
       <ButtonPanel />
     </div>
```

Comment:
The children inside `App.js` should be inside of a React.Fragment, not a div
![image](https://user-images.githubusercontent.com/73254040/98397355-dcc8f100-2024-11eb-8680-e82390e48fbd.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)